### PR TITLE
added menu to choose size of martix and number of deletions

### PR DESCRIPTION
**Pull request by yellowBunny**
hej Andrew. I have hope is working and you see anything. Sory for ma bad english :)

### CODE REVIEW

File: src_kolko_krzyżyk/TicTacToe_expand.py [hunk 1]
```diff
@@ -185,7 +185,35 @@ def main(self):
...
                 player = 0
 
-game = TicTacToe(5)
+def menu():
+    print('''\n\n\t"Simple game for two person in Tic Tac Toe.
+     Player decides how big the board is. Valid board (2 to n)
+     where board are matrix n x n. Good Luck and Have Fun."\n\n''')
+    # default matrix_choice
+    matrix_choice = 3
+    while True:
```
Comment (andrzejgorski) [inline src_kolko_krzyżyk/TicTacToe_expand.py:194]: You can move the logic of asking the user of choice to different function which could looks like that:
```python
    matrix_size = set_matrix_size()
```
where the content of this function could be:
```
def set_matric_size():
    while True:
        value = input(...)
        try:
            return int(value)
        except ValueError:
            print(...)
```

File: src_kolko_krzyżyk/TicTacToe_expand.py [hunk 2]
```diff
@@ -185,7 +185,35 @@ def main(self):
...
+        matrix_choice = input('Please enter size of martix from two to n: ')
+        try:
+            matrix_choice = int(matrix_choice)
+            break
+        except:
+            print("Dosen't entered number please try again" )
+
+    while True:
+        deletion_choice = input('Please enter number of deletions:'
```
Comment (andrzejgorski) [inline src_kolko_krzyżyk/TicTacToe_expand.py:203]: You can replace the phrase `number of deletions` with `winning condition number`. This name is more meaningful for me but you can try to convince me, that phrase is good enough.  You can also rename `deletion_choice` with `wining_condition`.

File: src_kolko_krzyżyk/TicTacToe_expand.py [hunk 3]
```diff
@@ -185,7 +185,35 @@ def main(self):
...
+    matrix_choice = 3
+    while True:
+        matrix_choice = input('Please enter size of martix from two to n: ')
+        try:
+            matrix_choice = int(matrix_choice)
+            break
+        except:
+            print("Dosen't entered number please try again" )
+
+    while True:
```
Comment (andrzejgorski) [inline src_kolko_krzyżyk/TicTacToe_expand.py:202]: As like above. You can move asking user of wining condition to a different function and place here code like:
```
wining_condition = set_wining_condition()
return TicTacToe(matrix_size, wining_condition).main()
```

File: src_kolko_krzyżyk/TicTacToe_expand.py [hunk 4]
```diff
@@ -185,7 +185,35 @@ def main(self):
...
+            if 1 <= deletion_choice <= matrix_choice:
+                return TicTacToe(matrix_choice, deletion_choice).main()
+            else:
+                print('Error deletions number')
+        except:
+            print("Dosen't entered number please try again")
+
+
+
+#main
```
Comment (andrzejgorski) [inline src_kolko_krzyżyk/TicTacToe_expand.py:216]: Remove this comment.

File: src_kolko_krzyżyk/TicTacToe_expand.py [hunk 5]
```diff
@@ -185,7 +185,35 @@ def main(self):
             else:
                 player = 0
 
-game = TicTacToe(5)
+def menu():
+    print('''\n\n\t"Simple game for two person in Tic Tac Toe.
+     Player decides how big the board is. Valid board (2 to n)
+     where board are matrix n x n. Good Luck and Have Fun."\n\n''')
+    # default matrix_choice
```
Comment (andrzejgorski) [inline src_kolko_krzyżyk/TicTacToe_expand.py:192]: Remove this comment and the line below. They don't do anything.

**Review by andrzejgorski — COMMENTED** [review]
I'm really glad that you send me to check small parts of code, because they are easier to read to reviewer.

File: src_kolko_krzyżyk/TicTacToe_expand.py (hunk 5 above)
Comment (yellowBunny) [reply src_kolko_krzyżyk/TicTacToe_expand.py:192]: thanks for hints. In next week i will try implement your solutions.
BTW maybe you don't know I wrote few new method earlier but i don't see them here in changes

**Pull request merged**